src/GitHubParserHook.php

```diff
@@ -43,14 +53,58 @@ private function setFields( ProcessingResult $result ) {
 		$this->fileName = $params['file']->getValue();
 		$this->repoName = $params['repo']->getValue();
 		$this->branchName = $params['branch']->getValue();
+
+		foreach ( self::$syntaxHighlightFields as $val ) {
+			if ( isset( $params[$val] ) ) {
+				$this->$val = $this->cleanField( $params[$val]->getValue() );
```

Comment:
> shit

This wording is a bit discouraging. Call it "magic" or so and everybody also knows what is meant from context.